app/grandchallenge/serving/views.py

```diff
@@ -74,3 +75,33 @@ def serve_submissions(request, *, submission_pk, **_):
         )
 
     raise PermissionDenied
+
+
+def serve_component_interface_value(
+    request, *, component_interface_value_pk, **_
+):
+    try:
+        # output should only be connected to a single job; throw error if not?
+        civ = (
+            ComponentInterfaceValue.objects.filter(
+                pk=component_interface_value_pk
+            )
+            .prefetch_related("algorithms_jobs_as_output")
+            .first()
+        )
+
+    except ComponentInterfaceValue.DoesNotExist:
+        raise Http404("No ComponentInterfaceValue found.")
+
+    if request.user.has_perm(
+        "view_job", civ.algorithms_jobs_as_output.first()
+    ):
+        create_download.apply_async(
```

Comment:
This will always fail as create_download does not take `component_interface_value_id` as a kwarg. We do not need to track downloads of these files IMO.

Reply:
ok, I'll remove the create_download. I did wonder about the serving of these files. When I access one of these files, I don't get a download option but rather the browser shows the json content. This exposes the minio url, not sure if that's ok

Reply:
Yes, that's perfect, in production this will be CloudFront. If you would like to download the file instead use the download attribute in the template https://www.w3schools.com/tags/att_a_download.asp